### ISSUE TITLE
Name mismatch in persisted state

### ISSUE THREAD

**Issue by tpoliaw** [chrisvest/xv#25]
Was this app previously called `xv`? It caught me by surprise that I had an `xv/xv.state` file in my `.config` directory but had nothing called xv on my machine. 

Would it be considered a breaking change to use `xxv/xxv,.state` for that file instead? It would potentially be better suited to somewhere like `~/.local/state`, `~/.cache` or `~/.local/share` instead of `.config` as well (or equivalent for other OSes).

**Comment by chrisvest** [comment]
> Was this app previously called xv?

Yes, it was. But it turned out there was an X Image Viewer app by the same name, so I renamed it.

> Would it be considered a breaking change to use xxv/xxv,.state for that file instead? It would potentially be better suited to somewhere like ~/.local/state, ~/.cache or ~/.local/share instead of .config as well (or equivalent for other OSes).

Not breaking with this version number. Could still look for both, but create the `xxv` one if neither are found.
I don't recall finding an authoritative source on what goes in what directories, so I just picked what seemed best at the time.

I can't work on this myself, for the time being. Best I can do is hand out commit rights.
